org.eclipse.lsp4j.websocket.jakarta/src/main/java/org/eclipse/lsp4j/websocket/jakarta/WebSocketEndpoint.java

```diff
@@ -0,0 +1,51 @@
+/******************************************************************************
+ * Copyright (c) 2021 TypeFox and others.
```

Comment:
for the copyright, should I put my company name instead?

Reply:
Sorry - I missed this question earlier. Especially when you consider the two commits together the files are not really new files at all, but just copies. I think the lines should be `Copyright (c) 2019, 2021[...]` to not lose the original copyright year. And no, you should not change company names, additional companies are covered by "and others."